[system-health] Convert to Python 3

jleveque opened the pull request:
**- Why I did it**

As part of moving all SONiC code from Python 2 (no longer supported) to Python 3. 

**- How I did it**

- Convert system-health scripts to Python 3
- Build and install system-health as a Python 3 wheel
- Also convert newlines from DOS to UNIX.

**- How to verify it**

Ensure system-health package builds, installs and works properly

**- Which release branch to backport (provide reason below if selected)**

<!--
- Note we only backport fixes to a release branch, *not* features!
- Please also provide a reason for the backporting below.
- e.g.
- [x] 202006
-->

- [ ] 201811
- [ ] 201911
- [ ] 202006

Fixes #6104

jleveque commented:
@Junchao-Mellanox: Please review.

Junchao-Mellanox commented:
See https://github.com/Azure/sonic-buildimage/blob/6c362a08e72e14da8880586f089d4860e786947c/src/system-health/scripts/healthd#L71, system health depends on platform API to set the chassis LED. It means that we need the platform API to support python3 first.

jleveque commented:
@Junchao-Mellanox: I believe currently only Mellanox supports the system-health daemon. I will set this back to draft until Mellanox has a Python 3 sonic-platform package installed. Please keep me posted.

Junchao-Mellanox commented:
Sure.

keboliu commented:
a preliminary check, seems that we still need some change in this here https://github.com/Azure/sonic-buildimage/blob/master/src/system-health/health_checker/utils.py#L11

keboliu commented:
in these two lines
https://github.com/Azure/sonic-buildimage/blob/master/src/system-health/health_checker/service_checker.py#L43
https://github.com/Azure/sonic-buildimage/blob/master/src/system-health/health_checker/service_checker.py#L48

need decode the 'output' before using :   output.decode('utf-8')

jleveque commented:
> in these two lines
> https://github.com/Azure/sonic-buildimage/blob/master/src/system-health/health_checker/service_checker.py#L43
> https://github.com/Azure/sonic-buildimage/blob/master/src/system-health/health_checker/service_checker.py#L48
> 
> need decode the 'output' before using : output.decode('utf-8')

Thanks for catching this! Rather than decoding, I added a `text=True` parameter to the `subprocess.Popen()` call. This is an alias of the `universal_newlines=True` parameter which is only available in Python 3. Since we no longer need to support Python 2 with this package, I used the new, shorter, more understandable alias. This will force the output to always be a string.

Also, note that I found that the files all had DOS line endings, so I changed them all to UNIX, which is why the diff is now much larger.

keboliu commented on src/system-health/health_checker/utils.py:
```diff
@@ -1,25 +1,25 @@
+    """
+    try:
+        process = subprocess.Popen(command, shell=True, universal_newlines=True, stdout=subprocess.PIPE)
+        return process.communicate()[0].encode('utf-8')
```
should remove "encode", previously I found that it needs to add "decode" when using the return value of this function is because of this.

jleveque replied:
Done. Thanks!

jleveque commented:
@keboliu, @Junchao-Mellanox: Are we good to merge this PR now? If so, please review again.

jleveque commented:
Retest vsimage please

jleveque commented:
Retest vsimage please

keboliu reviewed (approved):
LGTM
